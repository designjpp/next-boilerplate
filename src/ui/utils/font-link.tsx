/* eslint-disable @next/next/no-css-tags */
export const FontLink = () => (
	<>
		<link rel='preconnect' href='https://css.gstatic.com/' crossOrigin='true' />
		<link rel='preload' as='style' href='/css/font.css' />
		<link rel='stylesheet' href='/css/font.css' />
	</>
);
