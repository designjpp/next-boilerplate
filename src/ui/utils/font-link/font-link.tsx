import Head from 'next/head';

/* eslint-disable @next/next/no-css-tags */
export const FontLinkHead = () => (
	<Head>
		<link rel='preconnect' href='https://css.gstatic.com/' crossOrigin='true' />
		<link rel='preload' as='style' href='/css/font.css' />
		<link rel='stylesheet' href='/css/font.css' />
	</Head>
);
