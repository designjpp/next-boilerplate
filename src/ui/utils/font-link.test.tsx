import { create } from 'react-test-renderer';

import { FontLink } from './font-link';

describe('<FontLink>', () => {
	it("should matches it's snapshots", () => {
		const fontLinks = create(<FontLink />).toJSON();
		expect(fontLinks).toMatchSnapshot();
	});
});
