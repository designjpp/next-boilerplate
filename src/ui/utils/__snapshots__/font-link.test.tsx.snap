// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FontLinks> should matches it's snapshots 1`] = `
Array [
  <link
    crossOrigin="true"
    href="https://css.gstatic.com/"
    rel="preconnect"
  />,
  <link
    as="style"
    href="/css/font.css"
    rel="preload"
  />,
  <link
    href="/css/font.css"
    rel="stylesheet"
  />,
]
`;
